terratorch/models/pixel_wise_model.py

```diff
@@ -70,16 +70,24 @@ def __init__(
         self.neck = neck
         self.rescale = rescale
 
+        # TODO Maybe it's better to pass it an input argument
```

Comment:
Agree. This code is super specific to a single model, passing a `patch_size` would help a lot.
Alternatively you could iterate over all named modules and look for a `patch_embed`, then check if there is a attribute `patch_size`, then if it is a single int value or a list.